src/infra/src/storage/remote.rs

```diff
@@ -240,6 +247,9 @@ fn init_aws_config() -> object_store::Result<object_store::aws::AmazonS3> {
     if cfg.s3.feature_http2_only {
         opts = opts.with_http2_only();
     }
+    if cfg.s3.max_idle_per_host >= 0 {
+        opts = opts.with_pool_max_idle_per_host(cfg.s3.max_idle_per_host)
```

Comment:
https://docs.rs/object_store/latest/object_store/struct.ClientOptions.html#method.with_pool_max_idle_per_host
it looks here we need `usize`, and use `-1` as default value is strange, the default value is `0`, no limit.

Reply:
Done!